rocketchatctl

```diff
@@ -824,11 +868,34 @@ do_upgrade_rocketchatctl(){
     fi
 }
 
+do_configure(){
```

Comment:
Looks like this handles systemd but what if they have traefik or caddy installed? Anything we can do to help switch that over?

Reply:
I added the option configure for traefik and caddy too, `configure --lets-encrypt`, and configure became `configure --rocketchat`, but I could also just keep configure for rocketchat and add the code I made for the lets-encrypt flag in the original configure